deploy/helm/infinibox-csi-driver/values.yaml

```diff
@@ -23,7 +23,7 @@ images:
   provisionersidecar: quay.io/k8scsi/csi-provisioner:v1.4.0
 
   # "images.snapshotter-sidercar" defines the container image used for the csi snapshotter sidercar
-  snapshottersidecar: quay.io/k8scsi/csi-snapshotter:v1.2.2
+  snapshottersidecar: quay.io/k8scsi/csi-snapshotter:v2.0.1
```

Comment:
Why are you changing this version. Is it needed for treeQ.

Reply:
changes reverted . it's not required for treeq